src/System.Diagnostics.Process/ref/System.Diagnostics.Process.Manual.cs

```diff
@@ -10,8 +10,18 @@ namespace System.Diagnostics
     public partial class Process : System.IDisposable
     {
         public void Dispose() { }
+        /// <summary>
+        /// Release all resources used by this process.
+        /// </summary>
+        /// <param name="disposing">
+        /// true to release both managed and unmanaged resources; false to release only unmanaged resources.
```

Comment:
This is what the .NET 4.6.2 intellisense files we're importing from had.  We could consider going over these again the use langwords, though.  @mairaw, have you noticed the usage (or lack of usage) of langwords?